skopt/tests/test_optimizer.py

```diff
@@ -194,3 +196,21 @@ def test_optimizer_base_estimator_string_smoke(base_estimator):
     opt = Optimizer([(-2.0, 2.0)], base_estimator=base_estimator,
                     n_initial_points=1, acq_func="EI")
     opt.run(func=lambda x: x[0]**2, n_iter=3)
+
+
+def test_defaults_are_equivalent():
```

Comment:
Another suggestion: consider generating random seed at the beginning of the test, and then feeding  it as seed. This would ensure that for different seeds optimizers act same. More like a paranoia, but you never know.